### ISSUE TITLE
EC-Earth3-Veg e2e DTR run fails at validate-biascorrected

### ISSUE THREAD

**Issue by emileten** [ClimateImpactLab/downscaleCMIP6#468]
Run : https://argo.cildc6.org/archived-workflows/default/ae3327f2-e46a-4908-a661-d049677a69cf

Log : 

```
Validating gs://biascorrected-492e989a/stage/CMIP/EC-Earth-Consortium/EC-Earth3-Veg/historical/r1i1p1f1/day/dtr/gr/v20211221191100.zarr
/opt/conda/lib/python3.9/site-packages/dask/utils.py:35: RuntimeWarning: All-NaN slice encountered
  return func(*args, **kwargs)
/opt/conda/lib/python3.9/site-packages/dask/core.py:119: RuntimeWarning: All-NaN slice encountered
  return func(*(_execute_task(a, cache) for a in args))
/opt/conda/lib/python3.9/site-packages/dask/utils.py:35: RuntimeWarning: All-NaN slice encountered
  return func(*args, **kwargs)
```

and so on until : 

```
Traceback (most recent call last):
  File "/argo/staging/script", line 47, in <module>
    tasks = dask.compute(*tasks)
  File "/opt/conda/lib/python3.9/site-packages/dask/base.py", line 570, in compute
    results = schedule(dsk, keys, **kwargs)
  File "/opt/conda/lib/python3.9/site-packages/dask/threaded.py", line 79, in get
    results = get_async(
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 507, in get_async
    raise_exception(exc, tb)
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 315, in reraise
    raise exc
  File "/opt/conda/lib/python3.9/site-packages/dask/local.py", line 220, in execute_task
    result = _execute_task(task, data)
  File "/opt/conda/lib/python3.9/site-packages/dask/core.py", line 119, in _execute_task
    return func(*(_execute_task(a, cache) for a in args))
  File "/argo/staging/script", line 34, in clear_memory_intensive_tests
    _test_dtr_range(d, v, data_type)
  File "/opt/dodola/dodola/core.py", line 758, in _test_dtr_range
    assert (
AssertionError: diurnal temperature range max is <xarray.DataArray 'dtr' ()>
dask.array<nanmax-aggregate, shape=(), dtype=float32, chunksize=(), chunktype=numpy.ndarray> for non-polar regions
```

Blocks progress on #265

**Comment by emileten** [comment]
@dgergel from this log and a quick check in the data : 



(1) The maximum non-polar DTR is `74.6025`, above the current threshold in the validation. 

(2) I think we need to add a `.values.item()` after the calculation of unique values that might be printed. 

(3) I am wondering where this bunch of `NaN` warning is coming from. It's not from the `dtr_range` checks. Looking at it.

**Comment by emileten** [comment]
I failed figuring out where (3) comes from. Dask is warning us about all nan slices, but I don't find any nan in the data, and the nan tests passed. I moved on since it's not blocking the run...

Regarding (2), I figured there are only 3 data points where the `<70` test does not pass, and the three values are all within 70-80 dtr. The three data points are all at latitude `-59.5`, so that's above the edge of our southern polar tests. 

I suggest we slightly shift up that edge ? 


```
high = ds['dtr'].where((ds.lat > -60) & (ds.lat < 60)).where(cond=ds.dtr>70, drop=True).compute()
np.argwhere(high.values>70)
array([[ 4,  5, 12],
       [ 4,  5, 13],
       [ 4,  5, 14]])
high.lat.values[5]
-59.5
```
